server/opts.go

```diff
@@ -1678,11 +1691,54 @@ func parseExportStreamOrService(v interface{}, errors, warnings *[]error) (*expo
 				*errors = append(*errors, err)
 			}
 		}
-
 	}
 	return curStream, curService, nil
 }
 
+// parseServiceLatency returns a latency config block.
+func parseServiceLatency(v interface{}) (serviceLatency, error) {
+	latency, ok := v.(map[string]interface{})
```

Comment:
Maybe support simple version too. 

```
latency: <results.service.latency>
```

Which will default to 100 for sampling.

Reply:
Ah, sorry. What does this mean? Is this an alternative config format than
```
    exports = [
      { service: "nats.add"
        response: singleton
        latency: {
          sample: 100%
          subject: "latency.tracking.add"
        }
        accounts: [cncf]
      }
    ]
```
¿?

Reply:
Ooh, based on your other comment, I think the answer is yes. Like this:
```
    exports = [
      { service: "nats.add"
        response: singleton
        latency: "latency.tracking.add"
        accounts: [cncf]
      }
    ]
```

Reply:
Yes correct.